### PR TITLE
[tf]Add sota_checkpoints tf test

### PR DESCRIPTION
**Pull request by daniil-lyakhov**

### CODE REVIEW

**Comment by daniil-lyakhov** [comment]
I should resolve this conversation before merge: https://github.com/openvinotoolkit/nncf/pull/445

**Review by alexsu52 — CHANGES_REQUESTED** [review]
Please, rebase your branch.

File: beta/tests/test_sota_checkpoints.py [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+}
+
+DATASET_TYPE_AVAILABILITY = {
+    "classification": True,
+    "object_detection": True,
+    "segmentation": False,
+}
+
+'''
+OPENVINO_DIR = PROJECT_ROOT.parent / 'intel' / 'openvino'
```
Comment (alexsu52) [inline beta/tests/test_sota_checkpoints.py:48]: Do you really need OpenVINO to validate checkpoints?

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: beta/tests/test_sota_checkpoints.py (hunk 1 above)
Comment (daniil-lyakhov) [reply beta/tests/test_sota_checkpoints.py:48]: Not really, it's part of pytorch sota_test_checkpoints paths. I thought it'll help in future. I'll remove it

**Comment by daniil-lyakhov** [comment]
There is some changes in pytorch sota_test_checkpoints: 
https://github.com/openvinotoolkit/nncf/commit/7af05ce7307ec0d58a0e6892fe880f33086c6603
Do I need to add this changes to our pr or current result table is ok and @pfinashx will add it in future?

**Comment by pfinashx** [comment]
> There is some changes in pytorch sota_test_checkpoints:
> [7af05ce](https://github.com/openvinotoolkit/nncf/commit/7af05ce7307ec0d58a0e6892fe880f33086c6603)
> Do I need to add this changes to our pr or current result table is ok and @pfinashx will add it in future?

Yes, you can add this changes, they modified reference FP32 metrics in result table.

**Comment by daniil-lyakhov** [comment]
https://github.com/openvinotoolkit/nncf/pull/445 was merged thus --metrics-dump-path param approved

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**